figures/label_sampling_experiment/plot_scripts/class_accuracy_figure_generator.py

```diff
@@ -0,0 +1,123 @@
+import os
+import glob
+from collections import OrderedDict
+import pandas as pd
+import seaborn as sns
+import matplotlib.pyplot as plt
+import pdb
+
+plt.switch_backend('agg')
+
+def get_dataframes(result_dir, file_path):
+    return {
+        # Get the head word of each file that will be parsed
+        os.path.splitext(os.path.basename(file))[0].split("_")[0]:
+        pd.read_csv(file, sep="\t")
+        .assign(num_lfs=lambda x: x['num_lfs'].map(lambda y: y if y != 'baseline' else 'BL'))
+        .query("num_lfs in [0,1,6,11,16,20,28,22,30] or num_lfs in [0,1,33,65,97,100]")
```

Comment:
Alex's question is good. Also the numbers are not ascending, not that evenly spaced, and more of an issue is that there's a lot of magic numbers here. Is there a way to do this that doesn't have these properties? I'm imagining 0 is a special case because it's the knowledge bases, but can the rest be generated and selected programmatically?

Reply:
Ack magic numbers.. Good point. Updated the code to give the reader more information about these numbers.